src/types/pathways-model.d.ts

```diff
@@ -8,7 +8,7 @@ declare module 'pathways-model' {
     id: string;
     name: string;
     description: string;
-    library: string;
+    library: string | string[];
```

Comment:
Is there a reason this was made to be an array? I can't find anywhere it is being used as an array and I'm worried it may have some negative effects on the CPG exporter which only allows a single library to be referenced here

Reply:
Turned this into an array so that all the CQL Libraries can be exported too.  If a pathway uses the "TNM" and "Tumor_Markers" Criteria Libraries then they'll be exported in with the pathway in this array.

Do you have a suggestion for how to include them otherwise?  It seems like the model (at least the exported representation) will need to change somehow.

Reply:
Rather than either type, I'd suggest we just make it an array. IMO it's easier to deal with an array of 1 entry rather than check if it's an array or string everywhere.

Reply:
Made library just an array in 6d7ea8b.